contracts/strategies/stkCvxCrv/harvester/Harvester.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "../../../../interfaces/IVaultRewardHandler.sol";
+import "../../../../interfaces/ICurvePool.sol";
+import "../../../../interfaces/ICurveTriCrypto.sol";
+import "../../../../interfaces/ICurveV2Pool.sol";
+import "../../../../interfaces/ICvxCrvDeposit.sol";
+import "../../../../interfaces/ICurveFactoryPool.sol";
+
+contract stkCvxCrvHarvester {
+    using SafeERC20 for IERC20;
+    address public owner;
+    address public immutable strategy;
+    uint256 public allowedSlippage = 9500;
+    uint256 public constant DECIMALS = 10000;
+    address public pendingOwner;
+
+    bool public useOracle = true;
```

Comment:
Shouldn't we start with `false`? I assume we start with a permissioned both that uses a `minAmount` for `harvestAndSetRewardWeight`. These oracle calcs would just cost meaningless gas I think.

Reply:
i don't really expect us to ever need to use setRewardWeight because 3CRV apr will always be lower. so we can have open harvest with the oracle calcs to avoid sandwiches.